dynamic-demo-plugin/locales/en/plugin__console-demo-plugin.json

```diff
@@ -1,7 +1,8 @@
 {
   "{{count}} Worker Node": "{{count}} Worker Node",
-  "{{count}} Worker Node_plural": "{{count}} Worker Nodes",
+  "{{count}} Worker Node_plural": "{{count}} Worker Node",
```

Comment:
Not sure what is going on with the latest i18n stuff -- but was this intended to be changed in your PR? Seems unrelated.

Reply:
@andrewballantyne hmm interesting. I did run 'yarn i18n' I wonder if this was part of the output of that command...i can def remove that for now though.

Reply:
Ah -- if this was done by the i18n tool -- not sure you'll be able to remove it. Must be a latent change from the i18n upgrade.